src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/renderPaymentMethod.js

```diff
@@ -10,13 +10,40 @@ function getFallback(paymentMethod) {
   return fallback[paymentMethod];
 }
 
+function getPersonalDetails() {
```

Comment:
are there cases/PMs where we have to pass a state along with the address (U.S. states)?